CMakeLists.txt

```diff
@@ -46,6 +47,8 @@ include_directories(${URLDISPATCHER_INCLUDE_DIRS})
 set(CC_WARNING_ARGS " -Wall -pedantic -Wextra -Wno-missing-field-initializers")
 
 add_subdirectory(src)
+add_subdirectory(data)
```

Comment:
This should go with the commit that adds the CMakeLists.txt file and the .xml file.